### PR TITLE
Updating Databricks getting started for 0.3 release

### PR DESCRIPTION
**Pull request by sameerz**
This closes issue #1075 .  The docs/demo/Databricks/generate-init-script.ipynb will need to be updated once 0.3 is available in maven central.

### CODE REVIEW

File: docs/demo/Databricks/generate-init-script.ipynb [hunk 1]
```diff
@@ -1 +1 @@
...
+{"cells":[{"cell_type":"code","source":["dbutils.fs.mkdirs(\"dbfs:/databricks/init_scripts/\")\n \ndbutils.fs.put(\"/databricks/init_scripts/init.sh\",\"\"\"\n#!/bin/bash\nsudo wget -O /databricks/jars/rapids-4-spark_2.12-0.3.0.jar https://oss.sonatype.org/content/repositories/staging/com/nvidia/rapids-4-spark_2.12/0.3.0/rapids-4-spark_2.12-0.3.0.jar\nsudo wget -O /databricks/jars/cudf-0.17-cuda10-1.jar https://oss.sonatype.org/content/repositories/staging/ai/rapids/cudf/0.17/cudf-0.17-cuda10-1.jar\"\"\", True)"],"metadata":{},"outputs":[],"execution_count":1},{"cell_type":"code","source":["%sh\ncd ../../dbfs/databricks/init_scripts\npwd\nls -ltr\ncat init.sh"],"metadata":{},"outputs":[],"execution_count":2},{"cell_type":"code","source":[""],"metadata":{},"outputs":[],"execution_count":3}],"metadata":{"name":"generate-init-script","notebookId":2645746662301564},"nbformat":4,"nbformat_minor":0}
```
Comment (tgravescs) [inline docs/demo/Databricks/generate-init-script.ipynb:1]: this is staging url right, once we publish we need to update again?

Comment (sameerz) [reply docs/demo/Databricks/generate-init-script.ipynb:1]: Once we publish we will need to update again, that is correct.  I plan to update all docs one more time before the release to point to the released jars.

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by GaryShen2008** [comment]
build

**Comment by GaryShen2008** [comment]
The blossom CI seems always failed by  "Could not connect to premerge-test-jenkins-rapids-premerge-github-586-pbshd-19lk6 to send interrupt signal to process".
@pxLi  I think we need to spend some time to figure out the root cause. Is there any timeout setting not correct?

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
plz upmerge to latest branch-0.3 to get the HiveStringType fix, thx~

**Comment by sameerz** [comment]
build

**Pull request merged**